applicationinsights-android/src/main/java/com/microsoft/applicationinsights/library/ApplicationInsights.java

```diff
@@ -228,19 +263,19 @@ private void initializePipeline(Context context) {
             application = this.weakApplication.get();
         }
         TelemetryClient.initialize(!this.telemetryDisabled, application);
-        TelemetryClient.startAutoCollection(this.telemetryContext, this.config, !this.autoAppearanceDisabled, !this.autoPageViewsDisabled, !this.autoSessionManagementDisabled);
+        TelemetryClient.startAutoCollection(this.telemetryContext, this.configuration, !this.autoAppearanceDisabled, !this.autoPageViewsDisabled, !this.autoSessionManagementDisabled);
     }
 
     /**
      * Triggers persisting and if applicable sending of queued data
      * note: this will be called
-     * {@link com.microsoft.applicationinsights.library.config.ApplicationInsightsConfig#maxBatchIntervalMs} after
+     * {@link Configuration#maxBatchIntervalMs} after
      * tracking any telemetry so it is not necessary to call this in most cases.
      */
     public static void sendPendingData() {
```

Comment:
I know. But that method-name suggests a queue, which is a level of detail, that the dev shouldn't care. We did that on purpose ;)